internal/providers/util/config.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"github.com/coreos/ignition/config"
+	"github.com/coreos/ignition/config/types"
+	"github.com/coreos/ignition/config/validate/report"
+	"github.com/coreos/ignition/internal/log"
+)
+
+func ParseConfig(logger *log.Logger, rawConfig []byte) (types.Config, report.Report, error) {
+	logger.Debug("parsing config: %+v", rawConfig)
```

Comment:
I think this needs to read:

```go
logger.Debug("parsing config: %s", string(rawConfig))
```

Otherwise, you'll end up with an array of ascii codes.